pkg/resources/network_policy.go

```diff
@@ -92,18 +92,22 @@ func NetworkPolicy() *schema.Resource {
 		Description:   "Resource used to control network traffic. For more information, check an [official guide](https://docs.snowflake.com/en/user-guide/network-policies) on controlling network traffic with network policies.",
 
 		CustomizeDiff: customdiff.All(
+			// For now, allowed_network_rule_list and blocked_network_rule_list have to stay commented and the implementation
```

Comment:
hmmm, I think that I tested it somewhere, and even had to adjust the implementation because of that, that the order of suppressions is suppress on field -> customized diff.

Are you sure that this is not something else causing the problems here? With that line of argumentation, all the diff suppressed attributes should be problematic (e.g. all the enums where me allow lowercas for example - there are tests in warehouse for it).

Reply:
Okay, please change this comment and document the findings - now, the comment is misleading, because this is not the problem with diff suppression, but the problem in how in the SDK v2 diff suppression is handled for the subset of params (lists, sets, maps).

Reply:
👌 (Will be addressed in the next pr)